tests/jeos/rpi_wifi.pm

```diff
@@ -0,0 +1,39 @@
+# SUSE's openQA tests
+#
+# Copyright 2015-2021 SUSE LLC
+# SPDX-License-Identifier: FSFAP
+
+# Summary: Check RPi Wifi
```

Comment:
Could you give more details here, please?
Especially since it does not try to connect to any WiFi network.

Reply:
The idea is that the worker host (also a rpi) will span up a software wifi ap that the shelly powerplugs and the sut during this test module can connect to.
This is not yet implemented in the test as I'm busy setting up an actual rpi based worker right now (I was using my laptop as a worker before) so that I have a working wifi to connect to.

Reply:
Could you be more verbose than "Check RPi WiFi" please?
Also, please add a note about required vars: `RPI_WIFI_PSK` and `RPI_WIFI_WORKER_IP` AFAICS.

Reply:
done